app/layout.js

```diff
@@ -18,27 +19,20 @@ export default function RootLayout({ children }) {
     <html lang="en">
       <body className={`${inter.className} min-h-screen bg-background font-sans antialiased`}>
         {/* ThemeProvider+ DarkMode */}
-
         <ThemeProvider
           attribute="class"
           defaultTheme="system"
           enableSystem={true}
           disableTransitionOnChange
-          >
-        
-        {/* @todo - refactor this into  SiteHeader component*/}
-        <div className="relative flex min-h-screen flex-col">
-         <header className="sticky top-0 z-50 w-full border-b bg-background/95 backdrop-blur supports-[backdrop-filter]:bg-background/60">
-          <div className="container flex h-14 items-center">
-            <Navbar />
+        >
+          {/* @todo - refactor this into  SiteHeader component*/}
+          <div className="relative flex min-h-screen flex-col">
+            <Header />
+            <div className="flex-1">{children}</div>
+            <Footer></Footer>
           </div>
-         </header>
-
-         <div className="flex-1">{children}</div>
-        </div>
-
         </ThemeProvider>
-      </body>
-    </html>
+      </body >
+    </html >
```

Comment:
```suggestion
      </body>
    </html>
```

Seems like an unnecessary introduction of spaces that other components don't have?